plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -56,18 +56,20 @@
                                         </items>
                                     </items>
                                 </folder>
+                                <folder type="org.jkiss.dbeaver.ext.cubrid.model.CubridTrigger" label="%tree.triggers.node.name" icon="#folder_trigger" description="%tree.triggers.node.tip" visibleIf="object.supportsTrigger()">

```

Comment:
`visibleIf="object.supportsTrigger()"`
There is no need to add this condition if you already know that Cubrid supports triggers.